### PR TITLE
[14.0][IMP] oxigen_crm: do not show email and phone fields when using generic customer

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
@ForgeFlow

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Supersedes https://github.com/oxigensalud/odoo-addons/pull/85.

File: oxigen_crm/models/crm_lead.py [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+    )
+
+    @api.depends("partner_id")
+    def _compute_is_generic_customer(self):
+        generic_crm_customer_id = int(
             self.env["ir.config_parameter"].sudo().get_param("generic_crm_customer")
         )
-        if self.partner_id.id == generic_crm_customer:
+        for rec in self:
+            rec.is_generic_customer = rec.partner_id.id == generic_crm_customer_id
```
Comment (MiquelRForgeFlow) [inline oxigen_crm/models/crm_lead.py]:
```suggestion
            rec.is_generic_customer = rec.partner_id.id in generic_crm_customer_ids
```
what if in the future we have 2 generic customers?

Comment (LoisRForgeFlow) [reply oxigen_crm/models/crm_lead.py]: covered in a new commit

**Review by MiquelRForgeFlow — APPROVED** [review]
Squash?

**Comment by LoisRForgeFlow** [comment]
Tested locally with admin and demo users, with both list an integer parameters, merging.

**Pull request merged**